.github/workflows/update-dependencies.yml

```diff
@@ -0,0 +1,56 @@
+name: update-dependencies
+
+on:
+  repository_dispatch:
+    types: [update-dependency]
```

Comment:
This will be updated to allow a `manual` trigger both for testing purposes and in production.